ci_framework/playbooks/07-deploy-edpm-with-bmaas.yml

```diff
@@ -0,0 +1,58 @@
+- name: Run pre_bmaas hooks
+  vars:
+    hooks: "{{ pre_bmaas | default([]) }}"
+    step: pre_bmaas
+  ansible.builtin.import_playbook: ./hooks.yml
+
+- hosts: "{{ cifmw_target_host | default('localhost') }}"
+  gather_facts: false
+  tasks:
+    - name: Call make_edpm_compute_baremetal target
+      ansible.builtin.include_role:
+        name: 'install_yamls_makes'
+        tasks_from: 'make_edpm_compute_baremetal'
+      vars:
+        make_edpm_compute_baremetal_env:
+          NETWORK_ISOLATION: false
+          OPENSTACK_IMG: "{{ cifmw_operator_build_output['operators']['openstack-operator']['image_catalog'] }}"
+
+    - name: Generate Bash Script to check if compute nodes are provisioned
+      ansible.builtin.copy:
```

Comment:
This should really, really be ansible. I don't see what couldn't be done in plain ansible, allowing proper task decoupling, better error handling and, later/soon, proper custom module for the handling.

Reply:
moved to ansible.